lms/templates/courseware/course_updates.html

```diff
@@ -1,11 +1,10 @@
 <%! from django.utils.translation import ugettext as _ %>
 <section aria-labelledby="course-updates-heading">
-    <h2 class="hd hd-2 sr" id="course-updates-heading">${_('All course updates')}</h2>
```

Comment:
@cptvitamin Hey Mark - Following @cahrens comments, on tests failure, does it make sense to put it back or change tests to accomodate the UI change?